state/remoteapplication.go

```diff
@@ -249,6 +249,11 @@ func copyAttributes(values attributeMap) attributeMap {
 	return result
 }
 
+// DestroyOperation returns a model operation to destroy this remote application.
+func (s *RemoteApplication) DestroyOperation(force bool) *DestroyRemoteApplicationOperation {
```

Comment:
I do not think you need this one... Just use the one below.

(or rename the existing one)

Reply:
I like the idea of renaming, as RemoteApplication is implied as it's a method.

Reply:
Renamed.